react/global.css

```diff
@@ -1,64 +1,3 @@
-.vtex-product-kit-list__item-swap-button:hover div {
-  display: inline;
-}
-
-.vtex-product-kit-list .vtex-product-summary--normal {
-  width: 240px;
-}
-
-.vtex-product-kit-list .vtex-price-list {
-  font-size: .875rem;
-}
-
-.vtex-product-kit-list .vtex-price-selling {
-  font-size: 1.25rem;
-}
-
-.vtex-product-kit-list__details {
-  width: 240px;
-}
-
-.vtex-product-kit-list .slick-slider {
-  width: 100%;
-}
-
-/* Arrows */
-.vtex-product-kit-list .vtex-slider__arrow-left:before,
-.vtex-product-kit-list .vtex-slider__arrow-right:before {
-  font-size: 3vw;
-  color: black;
-}
-
-.vtex-product-kit-list .vtex-slider__arrow-left {
-  left: -4.2%;
-  z-index: 1;
-  top: 44%;
-}
-[dir='rtl'] .vtex-product-kit-list .vtex-slider__arrow-left {
-  left: auto;
-}
-
-.vtex-product-kit-list .vtex-slider__arrow-right {
-  top: 44%;
-}
-[dir='rtl'] .vtex-product-kit-list .vtex-slider__arrow-right {
-  right: auto;
-}
-
-/* Dots */
-.vtex-product-kit-list .vtex-slider__dots {
-  position: relative;
-  top: 5px;
-}
-
-.vtex-product-kit-list .vtex-slider__dots li button {
-  color: black;
-}
-
-.vtex-product-kit-list .vtex-slider__dots li button:before {
-  color: black;
-  font-size: 1vw;
-}
-.vtex-product-kit-list .vtex-slider__dots li.slick-active button:before {
-  color: black;
+.vtex-product-kit-1-x-listContainer .vtex-store-components-3-x-dots li button:before {	
```

Comment:
You should never make css rules using html elements, use classes instead (because it is faster to the browser apply rules of classes instead of html elements), so you should add a class to these `li` and `button`

Reply:
The li and button belong to a external lib, i don't think there is a way to what you are proposing.